### DISCUSSION TITLE
<title>Perdon por la ignorancia.. estoy aprendiendo.

### DISCUSSION THREAD

**Discussion by ramirox98** [community/community#41563] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

estoy aprendiendo c++ y se me ocurrio buscar un proyecto en Git Hub.. encontre varios pero solo son los archivos y no se como hacer para que corran... estoy seguro de que es super sencillo pero no se como.. todos tienen una especie de "COMANDO" que tiene este orden 

`"ejemplo " RELEASE C++,C / ejemplo / ejemplo2 / ejemplo3 / ejemplo 4 / ejemplo 5`

si alguien esta dispuesto a ayudarme le estoy eternamente agradecido.. llevo dias investigando y nada... Gracias!

**Comment by ivangongoram** [comment, marked as answer]
Hola @ramirox98,

Que bueno que estes aprendiendo c++, es la base para dominar cualquier lenguaje. Respecto a tu problema, me imagino que estas utilizando algun editor de codigo como: sublime text, o a lo mejor el block de notas XD. Lo que tienes que hacer es utilizar un compilador de codigo como Visual Studio, una vez lo tengas abres el ejemplo y lo ejecutas ▶️.

 Me cuentas como te fue 😃.

**Reply by ramirox98** [reply]
Hola Ivan,Gracias por responder! Ésto udamdo Visual Studio.El problema es que no se como correr el programa. Agradecería mucho si pudieras ayudarme con eso.

**Reply by jfchicaiza** [reply]
la forma mas fácil es ejecutarlo con comandos, para ello tiene que tener instalado el compilador oportuno. en este vídeo puedes ver como se hace, https://www.youtube.com/watch?v=8CNRX1Bk5sY. Te recomiendo que no dependas de un ide para  compilar los programas. Lo mejor es saber como se hace por comandos.
